phpBB/language/en/install.php

```diff
@@ -82,6 +82,9 @@
 	<p>Only those databases supported on your server will be displayed.',
 
 	'ACP_LINK'	=> 'Take me to <a href="%1$s">the ACP</a>',
+
+	'INSTALL_PHPBB_IS_ALREADY_INSTALLED'	=> 'phpBB is already installed.',
```

Comment:
How about just using INSTALL_PHPBB_ALREADY_INSTALLED and INSTALL_PHPBB_NOT_INSTALLED?
The "is" is kind of useless.